moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp

```diff
@@ -62,7 +62,7 @@ namespace moveit_rviz_plugin
 // ******************************************************************************************
 RobotStateDisplay::RobotStateDisplay() :
   Display(),
-  update_state_(false)
+  update_state_(false), load_robot_model_(false)
```

Comment:
this should be on a new line, or all three initializers should be on the same line